components/inbound-endpoints/org.wso2.carbon.inbound.endpoint/src/main/java/org/wso2/carbon/inbound/endpoint/internal/http/api/ConfigurationLoader.java

```diff
@@ -161,6 +160,49 @@ public static void loadInternalApis(String apiFilePath) {
         }
     }
 
+    private static void setHandlers(OMElement apiElement, InternalAPI api) {
```

Comment:
populateHandlers() is a better name that setHandlers()

Reply:
Fixed